command/sub8_launch/launch/subsystems/perception.launch

```diff
@@ -1,16 +1,21 @@
 <launch>
+  <arg name="orange_rectangle" default="True" />
+  <arg name="buoy" default="False" />
+
   <!-- roslaunch sub8_launch perception.launch color_calibration:=gazebo_color_calibration.yaml -->
   <arg name="color_calibration" default="default_color_calibration.yaml" />
 
   <rosparam file="$(find sub8_launch)/config/course_geometry.yaml" />
   <rosparam file="$(find sub8_launch)/config/$(arg color_calibration)" />
 
-  <node pkg="sub8_perception" type="buoy_2d.py" name="buoys" output="screen">
-    <rosparam file="$(find sub8_launch)/config/buoy_2d.yaml" command="load" ns="buoys" />
-  </node>
+  <group ns="vision">
+    <node if="$(arg buoy)" pkg="sub8_perception" type="buoy_2d.py" name="buoys" output="screen">
+      <rosparam file="$(find sub8_launch)/config/buoy_2d.yaml" command="load" ns="buoys" />
+    </node>
 
-  <node pkg="sub8_perception" type="path_marker_finder.py" name="path_marker_finder" output="screen">
-    <rosparam file="$(find sub8_launch)/config/path_marker_finder.yaml" command="load" />
-  </node>
+    <node if="$(arg orange_rectangle)" pkg="sub8_perception" type="orange_rectangle_finder.py" name="orange_rectangle">
+      <rosparam file="$(find sub8_launch)/config/orange_rectangle_finder.yaml" command="load" />
```

Comment:
One objective of this PR is to address #253, to make this perception node generic so it can be used to find the two challenges involving orange rectangles, the path marker and the lid of the bins challenge.